.github/workflows/pr_sync_ci.yml

```
@@ -40,10 +39,10 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
 
+      # run 명령어로 schema 만드는 명령어 실행
+      - name: Into Schema
+        run: mysql -h localhost -P 3306 -u ${{secrets.TEST_DB_USERNAME}} -p${{secrets.TEST_DB_PASSWORD}} --protocol=tcp < ./mysql/schema.sql
```

해당 job에서 실패하네요 아마도 아래와 같이 바꿔줘야 할 것 같아요! [stackoverflow](https://stackoverflow.com/questions/58222386/github-actions-using-mysql-service-throws-access-denied-for-user-rootlocalh)

```suggestion
      - name: Into Schema
        run: mysql -h 127.0.0.1 -P 3306 -u ${{secrets.TEST_DB_USERNAME}} -p${{secrets.TEST_DB_PASSWORD}} --protocol=tcp < ./mysql/schema.sql
```

---

오 이건 제 개인 repo에서는 성공했었는데 다시 봐야겠네요